.env.sample

```diff
@@ -1,5 +1,10 @@
-PANO_MINIO_URL=127.0.0.1:9000
-PANO_MINIO_ACCESS_KEY=zlGjx3eZm9JLEc39A87f
-PANO_MINIO_SECRET_KEY=yPhvV67qhUItegOSTdKbhX1eerZSHe86SciwJ6Td
-PANO_MINIO_BUCKET=panoramas
-PANO_MINIO_SECURE=False
+PG_CONN=postgresql://pano-dev:pano-dev@localhost:5432/pano-dev
```

Comment:
```suggestion
PG_CONN=postgresql+psycopg://pano-dev:pano-dev@localhost:5432/pano-dev
```

Reply:
I have not seen this URL format before and will need to look it up.

Reply:
https://stackoverflow.com/a/75885662

Reply:
That's my least favorite type of non-obvious behavior, but, I guess it makes sense in historical and backwards-compatibility context.